examples/vision/retinanet.py

```diff
@@ -0,0 +1,700 @@
+"""
+Title: Object Detection with RetinaNet
+Author: [Srihari Humbarwadi](https://twitter.com/srihari_rh)
+Date created: 2020/05/17
+Last modified: 2020/06/22
+Description: Implementation of Focal Loss for Dense Object Detection.
+"""
+import os
+import re
+import zipfile
+
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+
+import matplotlib.pyplot as plt
+import tensorflow_datasets as tfds
+
+"""
+## Download subset of COCO2017 dataset
+"""
+
+url = "https://github.com/srihari-humbarwadi/datasets/releases/download/v0.1.0/data.zip"
```

Comment:
Where is this subset being used?

Reply:
```
(train_dataset, val_dataset), dataset_info = tfds.load(
    "coco/2017", split=["train", "validation"], with_info=True, data_dir="data"
)
```
@tanzhenyu  The "data" dir that I download here, has only 1 train and 1 validation tfrecord from the `tensorflow_datasets`  coco files. Changing the `data_dir` arg while loading the dataset will now only load the tfrecords from the "data" dir. Setting this to `None` will download the entire coco dataset, which users can do if they are trying to train it on the entire dataset.